src/EditorFeatures/Core/Implementation/Diagnostics/DiagnosticsSquiggleTaggerProvider.cs

```diff
@@ -129,7 +135,7 @@ private string GetErrorTypeFromDiagnosticProperty(DiagnosticData diagnostic)
 
         private string GetErrorTypeFromDiagnosticTags(DiagnosticData diagnostic)
         {
-            if (diagnostic.CustomTags.Count <= 1)
+            if (diagnostic.CustomTags.Count < 1)
```

Comment:
Ah, you are right, thanks for the catch. I was trying something else and forgot to revert this.